src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/configuration/SwaggerConfig.java

```diff
@@ -26,8 +26,8 @@
 	@Value("${payment.service.api.version}")
 	private String version;
 	
-	@Value("${payment.service.environment}")
-	private String environment;
+	@Value("${payment.service.swagger.enabled}")
```

Comment:
Should we default it to false: `${payment.service.swagger.enabled:false}`

Reply:
@alexjoybc - We could, but then this allows values to be missing from the properties file. I prefer to have the application generate an error at runtime when a value is missing so it forcibly ensures that the all name values pairs in the property file have been included and set.

Reply:
Additionally it presents all possible name value pairs in the properties file and doesn't  leave some obfuscated in the application classes.